src/OSPSuite.Infrastructure.Import/Core/ColumnInfo.cs

```diff
@@ -64,9 +65,9 @@ public bool IsMeasurement
          get => !IsBase && !IsAuxiliary;
       }
 
-      public bool SupportsDimension(IDimension dimensionForUnit)
+      public IDimension DimensionForUnit(string unitName)
       {
-         return SupportedDimensions.Contains(dimensionForUnit);
+         return SupportedDimensions.DimensionForUnit(unitName);
```

Comment:
@msevestre I have removed the dependency to the DimensionFactory, since we have already the SupportedDimensions for the specific column, we do not need to search in all the Dimnsions in DimensionFactory for the correct one for this unit, and then check again that it is supported by the ColumnInfo - we just search directly in the SupportedDimensions. I think it is quite better a solution this way.